### DISCUSSION TITLE
SEO support

### DISCUSSION THREAD

**Discussion by n-igarashi** [monstar-lab-oss/ml-frontend#1] [General]
SSR or Dynamic Rendering

**Comment by mvn-luanvu-hn** [comment]
Hi @yukinoda @mvn-binhnguyen-hn
I think one of your project is using the dynamic rendering mechanism to support SEO
- if it is search bot, redirect to pre-rendered page
- It not, redirect to SPA

After redirecting to pre-rendered page, how do you detect the meaningful content is rendered?
For example, in the post page how do you detect the post is rendered?
I think we can check if a CSS selector exist or not
